### PR TITLE
[Relay][OP] MultiboxPrior

### PR DESCRIPTION
**Pull request by kevinthesun**

### CODE REVIEW

File: python/tvm/relay/op/vision/multibox.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    steps : Tuple of float
+        Priorbox step across y and x, -1 for auto calculation.
+
+    offsets : tuple of int
+        Priorbox center offsets, y and x respectively.
+
+    clip : boolean
+        Whether to clip out-of-boundary boxes.
+
```
Comment (slyubomirsky) [inline python/tvm/relay/op/vision/multibox.py]: `MultiBoxPriorAttr` above mention default values for many of the arguments. Could you ensure that this comment matches those?

File: tests/python/relay/test_op_level4.py [hunk 2]
```diff
@@ -124,6 +124,25 @@ def test_binary_broadcast():
         ftype = func.checked_type
         assert ftype.ret_type == relay.TensorType((5, 10, 4), "int32")
 
+def test_multibox_prior():
```
Comment (slyubomirsky) [inline tests/python/relay/test_op_level4.py]: Could you test that the default arguments work as well?

File: include/tvm/relay/attrs/vision.h [hunk 3]
```diff
@@ -12,6 +12,34 @@
 namespace tvm {
 namespace relay {
 
+/*! \brief Attributes used in multibox_prior operators */
+struct MultiBoxPriorAttr : public tvm::AttrsNode<MultiBoxPriorAttr> {
```
Comment (slyubomirsky) [inline include/tvm/relay/attrs/vision.h]: I think for consistency with other files, you should probably name it `MultiBoxPriorAttrs` (with the final "s")

**Comment by tqchen** [comment]
@zhreshold can you also help review this PR?

**Comment by zhreshold** [comment]
This PR LGTM.

BTW, I am more interested in a standalone box NMS op so that GluonCV models can be used directly. @kevinthesun Do you know the progress?

**Comment by kevinthesun** [comment]
NNVM BoxNMS op is already there. Topi version BoxNMS is done using hybrid frontend, and waiting for the ability to compile with hybrid frontend. @were Any update?

**Pull request merged**